platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfiguration.java

```diff
@@ -1855,6 +1858,19 @@ public ConfigurationRole getRoleAtCreation() {
         return roleAtCreation;
     }
 
+    private void assertNotDetachedExtensionDoingExtending(Iterable<Configuration> extendsFrom) {
+        if (isDetachedConfiguration()) {
+            String summarizedExtensionTargets = StreamSupport.stream(extendsFrom.spliterator(), false)
+                .map(c -> "'" + c.getName() + "'")
+                .collect(Collectors.joining(", "));
+            DeprecationLogger.deprecateAction(String.format("Calling extendsFrom on %s", this.getDisplayName()))
+                .withContext(String.format("This is a detached configurations and can not extend other configurations, in this case: %s.", summarizedExtensionTargets))
```

Comment:
```suggestion
                .withContext(String.format("Detached %s should not extend configuration(s): %s.", this.getDisplayName(), summarizedExtensionTargets))
```

This wording seemed a little odd to me

Reply:
Adjusted the wording a little